src/errors/InvalidElementForSlicingError.ts

```diff
@@ -0,0 +1,5 @@
+export class InvalidElementForSlicingError extends Error {
+  constructor(public path: string) {
+    super(`Cannot slice element '${path}' which is not an array or choice`);
```

Comment:
I think we could clarify this message a bit more.  Either:

> Cannot slice element '${path}' since it is not an array or choice

or spelling it out even more

> Cannot slice element '${path}' since FHIR only allows slicing on choice elements (e.g., value[x]) or elements with max > 1

Reply:
https://github.com/FHIR/sushi/pull/952/commits/6e644f6f9393105bf1b2b6925f49a939c47cbcd7